DatabaseLibrary/build.gradle

```diff
@@ -1,22 +1,8 @@
-apply plugin: 'com.android.library'
+apply plugin: 'java'
 
-android {
-    compileSdkVersion 22
-    buildToolsVersion "21.1.2"
-
-    defaultConfig {
-        minSdkVersion 8
-        targetSdkVersion 22
-    }
-
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
-        }
-    }
-}
+sourceCompatibility = 1.6
+targetCompatibility = 1.6
 
 dependencies {
-    compile 'com.android.support:support-v4:22.0.0'
+    compile 'com.google.android:android:4.1.1.4'
```

Comment:
In maven repositories, now the highest version of Google Android Library is 4.1.1.4 (API 16)
http://search.maven.org/#browse%7C-697723448
https://jcenter.bintray.com/com/google/android/android/